AUTHORS

```diff
@@ -15,6 +15,10 @@ Borg authors ("The Borg Collective")
 Retired
 ```````
 
+Authors listed here are not intended for support, please see the `support section`_ instead.
```

Comment:
I'ld rather refer to `E-Mail addresses` (contacting someone personally via private e-mail).

Also, I'ld put that phrase to the very top of the document, because even for non-retired contributors it is true that one better uses github or the mailing list rather than personal e-mail.

Reply:
done